docs/src/developer/changelog.md

```diff
@@ -78,10 +78,40 @@ which contain exactly the same content.
 ````
 
 [towncrier](https://github.com/twisted/towncrier) preserves multiple
-paragraphs and formatting in a news fragment file, however concise single
-paragraph entries are encouraged.
+paragraphs and the formatting within a news fragment file, however concise
+single paragraph entries are encouraged.
 
 ```{tip}
 Run `towncrier --draft` to render a preview of the news fragment files in the
 `changelog` directory.
 ```
+
+## {fa}`road` Changelog Quality Assurance
+
+Quality assurance of `changelog` contributions is performed by the
+[ci-changelog](https://github.com/bjlittle/geovista/blob/main/.github/workflows/ci-changelog.yml)
+{fab}`github` Action.
```

Comment:
Just curious, is there a reason you are using GitHubActions instead of [pre-commit hook](https://towncrier.readthedocs.io/en/stable/pre-commit.html)?

Reply:
@tkoyama010 Good question ... but essentially I wanted more flexibility and control.

The [towncrier check](https://towncrier.readthedocs.io/en/stable/cli.html#towncrier-check) behaviour is kinda ideal, but I didn't like it wrapped into the pre-commits i.e., if a contributor installs the pre-commits then they are forced to know the correct pull-request number before they raise the pull-request, and that's a race condition against other contributors. So using the GHA delays that decision in their workflow, as they can push the pull-request change, secure a pull-request number and then push a commit with the correct pull-request number on the changelog news fragment file.

The ability to label the pull-requests to enable and disable the QA workflow is really handy, and also the use of the label might be beneficial for historical reasons. Additionally, the logging on the GHA is also pretty handy to record what's happened.

Also, not all pull-requests require to be checked by default e.g., from some bot/ci-services or from the contributor themselves. You can tell the `pre-commit.ci` service to skip the ci from a pull-request comment or more specifically the whole ci or a ci task in the `.pre-commit-config.yaml` but I didn't consider it ideal.

So all said, I think it's okay to take on the maintenance burden (which is small) of a bespoke GHA action rather than the `towncrier` pre-commit hook.